lib/Dialect/Torch/Transforms/RefineTypes.cpp

```diff
@@ -1364,6 +1369,18 @@ ChangeResult TypeAnalyzer::visitAtenMatmulOp(
   return getLatticeElement(op->getResult(0)).join(knowledge);
 }
 
+ChangeResult TypeAnalyzer::visitAtenMeanOp(
```

Comment:
Remember that you can use the 

https://github.com/llvm/torch-mlir/blob/0a5a38e898af4c9af995ef99939a77f727ee2270/lib/Dialect/Torch/Transforms/RefineTypes.cpp#L926

since `mean` is a reduction operation.

Reply:
Okay. Do you think the error that I have posted above is because of that?

Reply:
No, this should not affect anything. I commented here https://github.com/llvm/torch-mlir/pull/430#discussion_r754491674 on what I think the issue is :)